src/main/java/org/broadinstitute/hellbender/engine/AssemblyRegion.java

```diff
@@ -452,7 +453,7 @@ public SimpleInterval getReadSpanLoc() {
     /**
      * See {@link #getAssemblyRegionReference} with padding == 0
      */
-    public byte[] getAssemblyRegionReference( final IndexedFastaSequenceFile referenceReader ) {
+    public byte[] getAssemblyRegionReference( final ReferenceSequenceFile referenceReader ) {
```

Comment:
name of the `referenceReader` parameter seems weird given that the type is called `ReferenceSequenceFile`